### PR TITLE
Updated README.md

### LINKED ISSUE #3: README is missing examples of the snippets and their expansion.
I would like there to be a list within the README or another file (linked from the README) that is able to show all the possible snippet expansions.

### PR DESCRIPTION
**Pull request by username-is-already-taken2**
Hi there 

Is this what you had in mind for the README.md? 

I've tried to pull together various links and information etc. I've also created some place holders for the animated gifs

Let me know what you think and I'll amend where approprate.

Was trying to resolve
#3 & #2 

Best Regards

Gary

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,126 @@
...
+This extension for Visual Studio Code provides grammar and code snippets for InSpec when using [Visual Studio Code](http://code.visualstudio.com).
 
-> Tip: Many popular extensions utilize animations. This is an excellent way to show off your extension! We recommend short, focused animations that are easy to follow.
+![InSpec Extention Demo](/images/vscode-inspec-demo.gif)
 
-## Release Notes
+## Features
+#### Syntax/keyword highlighting:
+* InSpec - control blocks
+* InSpec - discribe blocks
```
Comment (burtlo) [inline README.md:10]: `describe`

**Comment by burtlo** [comment]
This is great!

**Comment by burtlo** [comment]
@username-is-already-taken2 I fixed a few of the typos and added the animated gif to get it started. I think it may be time to release a new version. Which I need to describe in the README or an additional file.

**Comment by username-is-already-taken2** [comment]
Thanks for the corrections,

I have no preference either way, I know some of the other (chef) projects I use seem to use a changelog rather than update the readme.  We update the readme internally but only because git renders it when we load the repo. :)

**Pull request merged**